PerfettoCds/Pipeline/Tables/PerfettoCpuSchedTable.cs

```diff
@@ -86,6 +118,14 @@ public static void BuildTable(ITableBuilder tableBuilder, IDataExtensionRetrieva
             tableGenerator.AddColumn(PriorityColumn, baseProjection.Compose(x => x.Priority));
             tableGenerator.AddColumn(StartTimestampColumn, startProjection);
             tableGenerator.AddColumn(EndTimestampColumn, endProjection);
+            tableGenerator.AddColumn(WakeEventFoundColumn, baseProjection.Compose(x => x.WakeEvent != null));
+            tableGenerator.AddColumn(WakerProcessNameColumn, baseProjection.Compose(x => x.WakeEvent?.WakerProcessName ?? String.Empty));
+            tableGenerator.AddColumn(WakerThreadNameColumn, baseProjection.Compose(x => x.WakeEvent?.WakerThreadName ?? String.Empty));
+            tableGenerator.AddColumn(WakerTidColumn, baseProjection.Compose(x => x.WakeEvent?.WakerTid ?? -1));
+            tableGenerator.AddColumn(WakerPriorityColumn, baseProjection.Compose(x => x.WakeEvent?.Priority ?? -1));
+            tableGenerator.AddColumn(WakerCpuColumn, baseProjection.Compose(x => x.WakeEvent != null ? (int)x.WakeEvent.Value.Cpu : -1));
+            tableGenerator.AddColumn(WakeTimestampColumn, baseProjection.Compose(x => x.WakeEvent?.Timestamp ?? Timestamp.MinValue));
```

Comment:
I know it's scope creep (maybe another PR), but it would be awesome if we could include WaitTime for thread as well - as that's something we use a lot to tell the threads that have been blocked the longest.

Reply:
This needs to have access to last CSwitch of the thread. As talked about in the meeting, I need to add it for critical path analysis. So, I will do it in next PR. Thanks.